doc/whats_new/v0.16.rst

```diff
@@ -118,6 +118,9 @@ New features
 - Added :func:`metrics.median_absolute_error`, a robust metric.
   By `Gael Varoquaux`_ and :user:`Florian Wilhelm <FlorianWilhelm>`.
 
+- Added :func:`metrics.mean_absolute_percentage_error`.
+  By :user:`Ashutosh Hathidara <ashutosh1919>`.
```

Comment:
The next release is going 0.23. So the changelog entry should be added in v0.23.rst instead.

Reply:
done